### ISSUE TITLE
[Enhancement] Add Support for Simple Corn

### ISSUE THREAD

**Issue by Insane96** [SirStendec/WirelessUtilities#29]
Right now it harvests the left one instead of the right one.
![2019-05-14_09 34 49](https://user-images.githubusercontent.com/7185104/57679444-a24ae780-762b-11e9-8cc3-c3dd525df6d8.png)

**Comment by SirStendec** [comment]
Added support for this in f39b098d0246e550a6d5e17ef5d51539dbe2b9f2. It isn't perfect because that mod doesn't drop items in a standard way, so some items will fall into the world. It is breaking them when correctly fully grown, however.

I ended up refactoring a bit, and adding CraftTweaker support for defining problem plants like this. (Still need to write the documentation for that.) That looks like: `mods.wirelessutils.crop_behaviors.addTallCrop(int priority, Block[] blocks);`

So like:
```
mods.wirelessutils.crop_behaviors.addTallCrop(1, [
    <simplecorn:corn>,
    <simplecorn:corn_mid>,
    <simplecorn:corn_top>
]);
```

(To clarify, support is built-in, the CraftTweaker stuff can just be used for *other* problem crops without needing the mod itself to have support.)
